api-common/src/main/kotlin/io/qalipsis/api/meters/CampaignMeterRegistry.kt

```diff
@@ -23,34 +23,70 @@ import java.util.function.ToDoubleFunction
 /**
  * Campaign lifecycle relevant meter registry.
  *
- * For the documentation, consult [MeterRegistry]
- *
  * @author Eric Jessé
  *
  */
 interface CampaignMeterRegistry {
 
+    /**
+     * Creates a new [Counter] metric to be added to the registry. This metric measures the
+     * count of specific events collected over time.
+     *
+     * @param scenarioName the name of the scenario under which the count is collected
+     * @param stepName the name of a step within the scenario
+     * @param name the name of the counter metric
+     * @param tags additional key-value pairs to associate with the counter metric
+     *
+     * @sample counterExample
+     */
     fun counter(
         scenarioName: ScenarioName = "",
         stepName: StepName = "",
         name: String,
         tags: Map<String, String> = emptyMap(),
     ): Counter
 
-    fun summary(
-        scenarioName: ScenarioName = "",
-        stepName: StepName = "",
-        name: String,
-        tags: Map<String, String> = emptyMap(),
-    ): DistributionSummary
-
+    /**
+     * Creates a new [Timer] metric to be added to the registry. This metric measures the duration of an operation or a task.
+     *
+     * @param scenarioName the name of the scenario under which the timer is recorded
+     * @param stepName the name of a step within the scenario
+     * @param name the name of the timer metric
+     * @param tags additional key-value pairs to associate with the timer metric
+     * @param percentiles a list of values within the range of 1.0-100.0, representing specific points of observation, defaults to a list of 50.0, 75.0 and 99.9
+     * @param histogramCounts a list of values, whose frequencies within a bucket/bin in a histogram are to be observed, defaults to an empty list
+     * @param minHistogramBoundary minimum boundary within which a histogram should be created, defaults to 10.0
```

Comment:
the default values here are not the same than the default values in the code. Furthermore, it would be great to explain how minHistogramBoundary and maxHistogramBoundary relate to histogramCounts. By reading the documentation, I do not understand how to use them.

Furthermore, does it make sense to use 10 and 1000 or 1000 and 10000 when not knowing what the timer deals with? Wouldn't it be better to just disable the histogram in that case?